_overviews/FAQ/index.md

```diff
@@ -196,18 +196,21 @@ be written using the `_` syntax.
 
 See also [SLS 6.23.2](https://scala-lang.org/files/archive/spec/2.13/06-expressions.html#placeholder-syntax-for-anonymous-functions).
 
-### Why doesn't my dotted expression work?
+### Why couldn't Scala infer the correct type in my code?
 
-When methods have type parameters, the compiler makes a best effort to infer them
-if they are not supplied explicitly in a method application.
-Type parameter inference is improved in Scala 3.
+It is difficult to generalize about type inference, because various features of the language
+affect how your code is construed. There may be several ways to rewrite your code to make
+the types fall out naturally.
 
-```
-scala> List("ab").toSet.head.head
-                             ^
-       error: value head is not a member of type parameter B
-```
-For more context, see [this section of the tour](https://docs.scala-lang.org/tour/multiple-parameter-lists.html#drive-type-inference).
+The most straightforward workaround is to supply explicit types in your code.
+
+That may involve specifying an explicit type to a definition, or a type argument to a method.
+
+Type inference is greatly improved in Scala 3. If Scala 2 doesn't compile your code, it's worth trying with Scala 3.
+
+Sometimes, using multiple parameter lists helps inference, as explained in [this section of the language tour](https://docs.scala-lang.org/tour/multiple-parameter-lists.html#drive-type-inference).
+
+For paradigmatic questions about type inference involving `toSet`, see the discussions on [this ticket](https://github.com/scala/bug/issues/7743) and a related [Q&A](https://stackoverflow.com/questions/5544536/in-scala-2-type-inference-fails-on-set-made-with-toset).
```

Comment:
overall LGTM but I ask for one change: s/paradigmatic/common/

Reply:
ok I accept pitching it to the commons